src/components/Contact.js

```diff
@@ -22,7 +22,7 @@ const Contact = observer(({assembly}) => (
           <li>{assembly.translate("contact.first")}</li>
         </List>
 
-        <Link href="https://github.com/uwcirg/tb-mobile-app" target="_blank">
+        <Link href="http://wa.me/5491127481963" target="_blank">
```

Comment:
Whose number is this?

Reply:
Is it something that you can put into the `.env` and `.sample.env` files, and then pull in with `process.env`?